panacea/src/pml_parser.yrl

```diff
@@ -127,10 +134,25 @@ action_attribute ->
 
 requires_expr ->
     drug '{' string '}'            : construct('$1', [], value_of('$3')).
-
+requires_expr ->
+    time '{' time_expr_list '}'    : construct('$1', [], value_of_time('$3')).
 requires_expr ->
     expression : '$1'.
 
+time_expr ->
+    years '{' number '}'           : construct('$1', [], value_of('$3')).
```

Comment:
The only thing that might cause a problem is introducing ambiguity in the parser. God speed.